daemon/logger/copier.go

```diff
@@ -23,6 +23,8 @@ type Copier struct {
 	copyJobs  sync.WaitGroup
 	closeOnce sync.Once
 	closed    chan struct{}
+	// err is the error occurs during `Copier.Run()`
+	err error
```

Comment:
Thank you for code review. :stuck_out_tongue_winking_eye: 
I changed it to be a map to avoid race. PTAL @AkihiroSuda